### PR TITLE
Change (un)install changes to only work for >57. Fixes #858 and Fixes…

### LINKED ISSUE #858: Confusing setting "Enable Container Tabs" under Privacy
With a brand new Firefox profile, if I go to Preferences, Privacy, I do not see an "Enable Container Tabs" checkbox.

Once I have installed the [Firefox Multi-Account Containers add-on](https://addons.mozilla.org/en-GB/firefox/addon/multi-account-containers/) from the add-ons directory, that setting appears and is checked by default. This checkbox seems redundant, after all, if the add-on is enabled, surely the feature of containers should be enabled as well.

Here is the most confusing part: if I disable the Firefox Multi-Account Containers add-on, containers contain to work, and the checkbox "Enable Container Tabs" continues to appear under Preferences, Privacy.

I would expect either the enabling of the add-on to control activation of containers, or the checkbox under Preferences, Privacy, but not a weird mixture of both.

### PR DESCRIPTION
**Pull request by jonathanKingston**
… #900 

@groovecoder - This code needs to be tested in 56 and 57 for upgrades/downgrades. Leaving the code here as a FYI. I think it is fine though.

### CODE REVIEW

File: bootstrap.js [hunk 1]
```diff
@@ -130,41 +130,58 @@ async function install() {
...
+  });
+}
+
+function checkLegacyFirefox() {
   const version = Services.appinfo.version;
   const versionMatch = version.match(/^([0-9]+)\./)[1];
-  if (versionMatch === "55"
-      || versionMatch === "56") {
+  if (Number(versionMatch) <= 56) {
+    return false;
```
Comment (groovecoder) [inline bootstrap.js]: This logic seems backwards, or the function name is confusing. "LegacyFirefox" would be <= 56?

Comment (jonathanKingston) [reply bootstrap.js]: Yeah sorry I renamed last minute, will work on this tomorrow.

**Comment by jonathanKingston** [comment]
@groovecoder I updated the code, I still need to check if this fixes the update issue I have been seeing in: #894 however I am worried that is a central issue still.

**Comment by groovecoder** [comment]
Need to remove all the .zip files from this PR.

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/multi-account-containers/pull/971 which removes and ignores the web-ext-artifacts zip files.

**Pull request closed without merging**